controllers/drplacementcontrol_controller_test.go

```diff
@@ -191,7 +191,7 @@ func getSyncDRPolicy() *rmn.DRPolicy {
 var drstate string
 
 // FakeProgressCallback of function type
-func FakeProgressCallback(drpcName string, state string) {
+func FakeProgressCallback(_ string, state string) {
```

Comment:
Can remove this parameter.

Reply:
It is related to: 
// ProgressCallback of function type
type ProgressCallback func(string, string),